clang/lib/CIR/CodeGen/CIRGenCall.cpp

```diff
@@ -177,16 +263,16 @@ RValue CIRGenFunction::emitCall(const CIRGenFunctionInfo &funcInfo,
   QualType retTy = funcInfo.getReturnType();
   const cir::ABIArgInfo &retInfo = funcInfo.getReturnInfo();
 
-  ClangToCIRArgMapping cirFuncArgs(cgm.getASTContext(), funcInfo);
+  ClangToCIRArgMapping cirFuncArgs(cgm.getASTContext(), funcInfo, false);
```

Comment:
I'm changing this function in https://github.com/llvm/llvm-project/pull/139159 to remove the ABI-specific handling.